sharding-core/sharding-core-optimize/src/main/java/org/apache/shardingsphere/core/optimize/sharding/segment/select/item/DerivedCommonSelectItem.java

```diff
@@ -29,7 +29,7 @@
 @ToString(callSuper = true)
 public final class DerivedCommonSelectItem extends CommonSelectItem {
     
-    public DerivedCommonSelectItem(final String expression, final String alias) {
-        super(expression, alias);
+    public DerivedCommonSelectItem(final String expression, final String alias, final String columnLabel) {
```

Comment:
How about this?
public DerivedCommonSelectItem(final String owner, final String name, final String alias)